traits/tests/test_trait_dict_list_set_event.py

```diff
@@ -0,0 +1,57 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" Test cases for TraitListEvent, TraitDictEvent, TraitSetEvent. """
+
+import unittest
+
+from traits.api import (
+    HasTraits, List, Dict, Set, on_trait_change,
```

Comment:
Not a big thing, but in new code we usually try to keep the list of imported names in alphabetical order (though we still argue with each other about how that applies to lowercase versus uppercase names).